### DISCUSSION TITLE
Initial C:N values for PPE calibration

### DISCUSSION THREAD

**Discussion by wwieder** [ESCOMP/CTSM#2317] [Ideas]
SI table 1 from [Zhu et al. 2020](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2019MS001841) has a synthesis of leaf, fine root and wood C:N ratios synthesized from TRY and FRED.  I wonder if we should use these as initial starting values for our PPE calibration for CLM6, @katiedagon @djk2120 @dlawrenncar @linniahawkins?

**Comment by linniahawkins** [comment]
This is great, thanks @wwieder – I've only checked leafcn but most of the Zhu et al., values are outside the ranges we set for the PPE. We will discuss how to update the defaults/ranges.

**Reply by wwieder** [reply]
The fine root and wood C:N ratios are pretty different, especially for shrubs!  Unfortunately, they don't provide uncertainty ranges for the PFT values provided in the Table to help constrain the PPE, but maybe the +/- 20% range is still appropriate.

Also curious, would this be helpful starting place for global FATES calibrations, @adrifoster @rosiealice or @ckoven ?

**Reply by katiedagon** [reply]
What's the CLM parameter for wood C:N? I'm just looking at the spreadsheet data on leafcn and frootcn (and adding this reference) and I can't find wood C:N.

**Reply by wwieder** [reply]
I think it's 500 for all woody pfts

**Reply by wwieder** [reply]
sorry 50 for live wood (sapwood, `livewdcn`) and 500 for dead wood (heart wood, `deadwdcn`).

**Reply by katiedagon** [reply]
Oh interesting, I do not think those are parameters we have captured in the PPE. But they are on the "more params" tab of the spreadsheet (not sure why they were excluded).
